### ISSUE TITLE
Question Tags for Screenreader

### ISSUE THREAD

**Issue by myzinsky** [DARC-e-V/50ohm#14]
Right now screen readers read correct answers with "Check" and wrong answers with "Clear". This should be change to "Richtig" und "Falsch"

Zammad context: #20364118
